src/Symfony/Component/Intl/Languages.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl;
+
+use Symfony\Component\Intl\Exception\MissingResourceException;
+
+/**
+ * Gives access to language-related ICU data.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Languages extends AbstractStaticBundle
+{
+    /**
+     * Returns all available languages.
+     *
+     * Languages are returned as lowercase ISO 639-1 two-letter language codes.
+     * For languages that don't have a two-letter code, the ISO 639-2
+     * three-letter code is used instead.
+     *
+     * A full table of ISO 639 language codes can be found here:
+     * http://www-01.sil.org/iso639-3/codes.asp
+     *
+     * @return string[] an array of canonical ISO 639 language codes
+     */
+    public static function getCodes(): array
```

Comment:
```suggestion
    public static function getLanguageCodes(): array
```

Same concern as above. [Language code](https://en.wikipedia.org/wiki/Language_code) is a term that commonly (in Symfony) refers to the ISO 2-letter code. The terms we establish here are:

* language code: ISO 2-letter code
* alpha 3 code: ISO 3-letter code